src/Core/AdminConsole/Models/Business/OrganizationDto.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Bit.Core.AdminConsole.Entities;
+using Bit.Core.Models.StaticStore;
+using Bit.Core.Utilities;
+
+namespace Bit.Core.AdminConsole.Models.Business;
+
+public record OrganizationDto(
+    Guid OrganizationId,
+    bool UseCustomPermissions,
+    int? Seats,
+    int? MaxAutoScaleSeats,
+    int? SmSeats,
+    int? SmMaxAutoScaleSeats,
+    Plan Plan,
+    string GatewayCustomerId,
+    string GatewaySubscriptionId,
+    bool UseSecretsManager
+)
+{
+    public static OrganizationDto FromOrganization(Organization organization) =>
+        new(organization.Id,
+            organization.UseCustomPermissions,
+            organization.Seats,
+            organization.MaxAutoscaleSeats,
+            organization.SmSeats,
+            organization.MaxAutoscaleSmSeats,
+            StaticStore.GetPlan(organization.PlanType),
+            organization.GatewayCustomerId,
+            organization.GatewaySubscriptionId,
+            organization.UseSecretsManager);
```

Comment:
Constructors are the standard C# way for creating objects so I think using a new constructor is more intuitive
```suggestion
    public OrganizationDto(Organization organization) 
        : this(
            OrganizationId: organization.Id,
            UseCustomPermissions: organization.UseCustomPermissions,
            Seats: organization.Seats,
            MaxAutoScaleSeats: organization.MaxAutoscaleSeats,
            SmSeats: organization.SmSeats,
            SmMaxAutoScaleSeats: organization.MaxAutoscaleSmSeats,
            Plan: StaticStore.GetPlan(organization.PlanType),
            GatewayCustomerId: organization.GatewayCustomerId,
            GatewaySubscriptionId: organization.GatewaySubscriptionId,
            UseSecretsManager: organization.UseSecretsManager)
    {
    }
```